### PR TITLE
remove aruba logo

### PR DESCRIPTION
**Pull request by britt6612**
[Deploy Preview](https://deploy-preview-INSERT_PR_#_HERE--keen-mayer-a86c8b.netlify.app/)

#### What does this PR do?
remove logo for Aruba 
#### What testing has been done on this PR?

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [ ] Small, medium, and large screen sizes
- [ ] Cross-browsers (FireFox, Chrome, and Safari)
- [ ] Light & dark modes
- [ ] All hyperlinks route properly

**Accessibility Checks**
- [ ] Keyboard interactions
- [ ] Screen reader experience
- [ ] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [ ] Console is free of warnings and errors
- [ ] Passes E2E commit checks
- [ ] Visual snapshots are reasonable

### CODE REVIEW

**Comment by britt6612** [comment]
See Slack thread for context 
https://hpe.slack.com/archives/C04LMJWQT/p1730391595489349

File: aries-site/src/pages/foundation/our-brand.mdx [hunk 1]
```diff
@@ -71,38 +65,7 @@ not veer from the layout and spacing of the elements.
...
-</UsageExample>
-
-### Aruba icon
+## HPE Aruba Networks
 
-Always use the Aruba icon at the top-left of your service or product
-experience. To ensure consistency across Aruba, the icon should always
-be accompanied by the company name and product/service name. Do not
-veer from the layout and spacing of the elements.
+For additional information and guidance on HPE Aruba Networking view [Brand Central](https://h10014.www1.hpe.com/brand-central/content/aruba).
```
Comment (jcfilben) [inline aries-site/src/pages/foundation/our-brand.mdx:70]: I'm getting an error when I go to this link

**Review by taysea — APPROVED** [review]
I'm fine with this for now given it's confirmed as outdated by Aruba and Chris. Can you reach out to Chris for him to clarify with the Aruba contact about what the proper content should be?

**Comment by britt6612** [comment]
> I'm fine with this for now given it's confirmed as outdated by Aruba and Chris. Can you reach out to Chris for him to clarify with the Aruba contact about what the proper content should be?

yes will set something to connect with Chris

**Review by jcfilben — APPROVED** [review]
Looks good!

**Pull request merged**